### PR TITLE
Milestone 2: Components

### PR DESCRIPTION
**Pull request by kenderb**
On this milestone I implemented:

- The directory structure.
- The App component.
- The Display component.
- The Button component.
- The ButtonPanel component

### CODE REVIEW

**Comment by sumancrest0001** [comment]
### Status: Changes Required :gear:

**Hello @kenderb**, You have done a good job and implemented all the requirements from the milestone-2. There is one change that needs to be made before moving to the next milestone. Let's have a look at it.

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  componentDidUpdate() {
+    this.onUpdate();
+  }
+
+  onUpdate() {
+    const { results } = this.props;
+    this.setState({ calculatorResults: results });
+  }
+
+  render() {
```
Comment (sumancrest0001) [inline src/components/Display.js]: **[Required]** Is there any particular reason why you made it a class component? It is not required, at least for now. Please make it functional.

**Comment by sumancrest0001** [comment]
**[Optional]** You can remote unnecessary logos from the public directory.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
**Please make the required changes and submit for another code review. 
If you have any queries or doubts regarding the review, message me on Slack (Suman Shrestha).**
**Happy coding !!!**

**Review by Torres-ssf — APPROVED** [review]
### Milestone approved :+1: 
Hi @kenderb :hand: 
Nice job on this milestone.
You may merge the PR.

Happy coding :computer: 
**Sergio Torres**

**Pull request merged**